#
# (c) Copyright 2015-2016 River Systems, all rights reserved.
#
# This is proprietary software, unauthorized distribution is not permitted.
#

srslib_framework/LogicalMetadata metadata

srslib_framework/LogicalCell[] data
srslib_framework/LogicalArea[] areas
